### ISSUE TITLE
Request to update connect popup messaging

### ISSUE THREAD

**Issue by johnbailon** [trezor/connect#800]
When initiating a `signMessage` command, the popup displays the following information:

<img width="640" alt="" src="https://user-images.githubusercontent.com/266389/115216628-43574c80-a137-11eb-8bdc-fa87ef885017.png">

```
- Read public keys from Trezor device
- Prepare Trezor device for transaction and data signing
```

Would it be possible to change the second line to only say _**Prepare Trezor device for message signing**_ or something to that effect? It would inform end users the actual scope of the permission they are allowing, which is to say, they are not going to sign any spend transactions.

**Comment by prusnak** [comment]
We'd need to split the `write` permission to `write-transaction` and `write-message` or something similar.

For Bitcoin this makes sense, but I am not sure about Ethereum and other networks, since smart contracts often transfer users' funds by signing a message (internal transfer).

**Comment by johnbailon** [comment]
> We'd need to split the `write` permission to `write-transaction` and `write-message` or something similar.
> 
> For Bitcoin this makes sense, but I am not sure about Ethereum and other networks, since smart contracts often transfer users' funds by signing a message (internal transfer).

Right. At least for Bitcoin, this would be highly appreciated. Happy to jump into contributing a pull request as well if needed. As for Ethereum, you're right, it might actually be abused so best to keep the message as is. Cheers!
